packages/atomic-layout/.github/README.md

```diff
@@ -0,0 +1 @@
+../../../README.md
```

Comment:
What does this do? Is it some GitHub feature?

Reply:
@kettanaito  yes, this is how we do it to show it github.